packages/klevu-ui/src/components/klevu-cms-list/klevu-cms-list.tsx

```diff
@@ -24,7 +24,7 @@ export class KlevuCmsList {
   /**
    * Caption of the listing
    */
-  @Prop() tCaption = getTranslation("cmsList.tCaption")
+  @Prop() tCaption: string = getTranslation("cmsList.tCaption")
```

Comment:
We should change the typing of `getTranslation()` to return string always. Then we do not need to set it to all properties.

Reply:
Updated the return type of getTranslation